.github/workflows/regression-report.yml

```diff
@@ -122,24 +180,227 @@ jobs:
           # shellcheck disable=SC2005
           echo "$(cat "$GITHUB_OUTPUT")"
 
+
+  handle_existing_config_bundle:
+    needs: [ parse_comment, checkout_and_cache ]
+    if: needs.parse_comment.outputs.mode == 'existing_config_bundle'
+    runs-on: ubuntu-latest
+    outputs:
+      config_link: ${{ steps.set_links.outputs.config_link }}
+      projects_link: ${{ steps.set_links.outputs.projects_link }}
+    steps:
+      - name: Restore Master Checkstyle Checkout
+        uses: actions/cache@v4
+        with:
+          path: .ci-temp/checkstyle
+          key: "${{ env.CACHE_KEY_MASTER }}"
+
+      - name: Restore PR Checkout
+        uses: actions/cache@v4
+        with:
+          path: .
+          key: "${{ env.CACHE_KEY_PR }}"
+
+      - name: Set config and projects links
+        id: set_links
+        run: |
+          BASE_URL="https://raw.githubusercontent.com/checkstyle/test-configs/main"
+          CONFIG_BUNDLE="${{ needs.parse_comment.outputs.config_bundle }}"
+          CONFIG_LINK="$BASE_URL/$CONFIG_BUNDLE/config.xml"
+          PROJECTS_LINK="$BASE_URL/$CONFIG_BUNDLE/list-of-projects.properties"
+          ./.ci/append-to-github-output.sh "config_link" "$CONFIG_LINK"
+          ./.ci/append-to-github-output.sh "projects_link" "$PROJECTS_LINK"
+
+  handle_generated_config_bundle:
+    needs: [ parse_comment, checkout_and_cache ]
+    if: needs.parse_comment.outputs.mode == 'generated_config_bundle'
+    runs-on: ubuntu-latest
+    outputs:
+      config_link: ${{ steps.generated_config_bundle.outputs.config_link }}
+      projects_link: ${{ steps.generated_config_bundle.outputs.projects_link }}
+    steps:
+      - name: Restore PR Checkout
+        uses: actions/cache@v4
+        with:
+          path: .
+          key: "${{ env.CACHE_KEY_PR }}"
+
+      - name: Find input file
+        id: find_input_file
+        run: |
+          INPUT_FILE="${{ needs.parse_comment.outputs.input_file }}"
+          WORKSPACE="${{ github.workspace }}"
+
+          FOUND_FILES=$(find "$WORKSPACE/src" -type f -path "*$INPUT_FILE")
+
+          if [ "$(echo "$FOUND_FILE" | wc -l)" -gt 1 ]; then
+            echo "Error: Multiple matching files found"
+            exit 1
+          elif [ -z "$FOUND_FILES" ]; then
+            echo "Error: Could not find input file"
+            exit 1
+          fi
+
+          FOUND_FILE="$FOUND_FILES"
+          ./.ci/append-to-github-output.sh "input_file" "$FOUND_FILE"
+
+      - name: Download DiffTool JAR
+        uses: robinraju/release-downloader@v1.8
+        with:
+          repository: "checkstyle/test-configs"
+          tag: "${{env.VERSION}}"
+          fileName: "extractor-all-2024-08-21.jar"
```

Comment:
```
          tag: "extractor-${{env.EXTRACTOR_VERSION}}"
          fileName: "extractor-all-${{env.EXTRACTOR_VERSION}}.jar"
```

Reply:
the version has mismatch, i already mentioend in discord. for now, i have used the env variable. please review.

Reply:
I will fix it when I be close to keyboard.

Reply:
fixed https://github.com/checkstyle/test-configs/releases/tag/extractor-2024-08-27